test/spi/spi_gaskets_test.dart

```diff
@@ -243,6 +243,167 @@ class SpiPairTest extends Test {
   }
 }
 
+class SpiMultiSubTest extends Test {
+  late final SpiInterface intfMain;
+  late final SpiInterface intfSubA;
```

Comment:
suggestion: might be better to make this a `List<SpiInterface>` to reduce the copy-paste for A,B,C,D?  And `List<SpiSub>` etc.